types/ext/dictionary-importer.d.ts

```diff
@@ -115,7 +115,7 @@ export type ArchiveFileMap = Map<string, ZipJS.Entry>;
 /**
  * A map of file types inside a dictionary and its corresponding regular expressions.
  */
-export type QueryDetails = Map<string, RegExp>;
+export type QueryDetails = [string, RegExp][];
```

Comment:
Name the array items.

Reply:
Do you mean something like this?
```ts
type QueryDetailsFileType = string;

type QueryDetailsRegularExpression = RegExp;

/**
 * A map of file types inside a dictionary and its corresponding regular expressions.
 */
export type QueryDetails = [QueryDetailsFileType, QueryDetailsRegularExpression][];
```
Hovering over QueryDetails won't give `[QueryDetailsFileType, QueryDetailsRegularExpression][]` but `[string, RegExp][]` because string and RegExp are atomic types